modules/release-notes/ref-release-notes-technology-preview.adoc

```diff
@@ -1,15 +1,108 @@
-However, these features are not fully supported under Red Hat Subscription Level Agreements, may not be functionally complete, and are not intended for production use.
-As Red Hat considers making future iterations of Technology Preview features generally available, we will attempt to resolve any issues that customers experience when using these features.
-See: link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview support scope].
-====
-
-[
+:_content-type: REFERENCE
+[id="technology-preview"]
+= Technology Preview
+
+This section lists Technology Preview features in {product} {product-version}.
+
+[IMPORTANT]
+====
+Technology Preview features provide early access to upcoming product innovations, enabling you to test functionality and provide feedback during the development process.
+However, these features are not fully supported under Red Hat Subscription Level Agreements, may not be functionally complete, and are not intended for production use.
+As Red Hat considers making future iterations of Technology Preview features generally available, we will attempt to resolve any issues that customers experience when using these features.
+See: link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview support scope].
+====
+
+[id="technology-preview-rhidp-5070"]
+== Version Compatibility Matrix
+
+The following tables list products and their versions compatible with {product-short} {product-version}:
+
+.Platform compatibility matrix
+[cols=2,%header]
+|===
+|Platform
+|Version
+
+| {ocp-brand-name}
+| {ocp-version-min}-{ocp-version}
+
+| {eks-brand-name} ({eks-short})
+| Supported for Kubernetes version {kubernetes-version}
+
+| {aks-brand-name} ({aks-short})
+| Supported for Kubernetes version {kubernetes-version}
+
+| {gcp-brand-name} ({gcp-short})
+|  Supported for Kubernetes version {kubernetes-version}
+|===
+
+.Product compatibility matrix
+[cols=2,%header]
+|===
+| Product
+| Version
+
+| Backstage
+| {product-backstage-version}
+|===
+
+.Frontend plugin packages compatibility matrix
+[cols=2,%header]
+|===
+| Package
+| Version
+
+| `@backstage/catalog-model`
+| 1.7.0
+
+| `@backstage/config`
+| 1.2.0
+
+| `@backstage/core-app-api`
+| 1.15.0
+
+| `@backstage/core-components`
+| 0.15.0
+
+| `@backstage/core-plugin-api`
+| 1.9.4
+
+| `@backstage/integration-react`
+| 1.1.32
+|===
+
+.Backend plugin packages compatibility matrix
+[cols=2,%header]
+|===
+| Package
+| Version
+
+| `@backstage/backend-app-api`
+| 1.0.0
+
+| `@backstage/backend-defaults`
+| 0.5.0
+
+| `@backstage/backend-dynamic-feature-service`
+| 0.4.1
+
+| `@backstage/backend-plugin-api`
+| 1.0.0
+
+| `@backstage/catalog-model`
+| 1.7.0
+
+| `@backstage/cli-node`
+| 0.2.8
+
+| `@backstage/config`
+| 1.2.0
+
+| `@backstage/config-loader`
+| 1.9.1
+|===
+.Additional resources
+* link:https://issues.redhat.com/browse/RHIDP-5070[RHIDP-5070]
+
+

```

Comment:
@jmagak There are new dynamic plugins included in RHDH 1.4, in previous RNs we listed these (see [here](https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html-single/release_notes/index#technology-preview-rhidp-3713))

As far as I know, we are adding the following plugins have been added in this release:

- Signals (frontend) - RH Tech Preview
- Notifications (backend) - RH Tech Preview
- Notifications (frontend) - RH Tech Preview
- Notifications Module Email - RH Tech Preview

@reestr @nickboldt @kadel Can you confirm if these should be mentioned in the RNs?